diff -up shaderc-2024.1/CMakeLists.txt.1~ shaderc-2024.1/CMakeLists.txt
--- shaderc-2024.1/CMakeLists.txt.1~	2024-04-30 20:23:34.000000000 +0200
+++ shaderc-2024.1/CMakeLists.txt	2024-07-14 03:16:51.497398679 +0200
@@ -120,7 +120,6 @@ endif(MSVC)
 
 # Configure subdirectories.
 # We depend on these for later projects, so they should come first.
-add_subdirectory(third_party)
 
 add_subdirectory(libshaderc_util)
 add_subdirectory(libshaderc)
@@ -129,11 +128,11 @@ if(${SHADERC_ENABLE_EXAMPLES})
     add_subdirectory(examples)
 endif()
 
-add_custom_target(build-version
-  ${Python_EXECUTABLE}
-  ${CMAKE_CURRENT_SOURCE_DIR}/utils/update_build_version.py
-  ${shaderc_SOURCE_DIR} ${spirv-tools_SOURCE_DIR} ${glslang_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/build-version.inc
-  COMMENT "Update build-version.inc in the Shaderc build directory (if necessary).")
+#add_custom_target(build-version
+#  ${Python_EXECUTABLE}
+#  ${CMAKE_CURRENT_SOURCE_DIR}/utils/update_build_version.py
+#  ${shaderc_SOURCE_DIR} ${spirv-tools_SOURCE_DIR} ${glslang_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/build-version.inc
+#  COMMENT "Update build-version.inc in the Shaderc build directory (if necessary).")
 
 function(define_pkg_config_file NAME LIBS)
   add_custom_target(${NAME}-pkg-config ALL
diff -up shaderc-2024.1/glslc/CMakeLists.txt.1~ shaderc-2024.1/glslc/CMakeLists.txt
diff -up shaderc-2024.1/libshaderc_util/CMakeLists.txt.1~ shaderc-2024.1/libshaderc_util/CMakeLists.txt
