diff -up shaderc-2020.4/glslc/CMakeLists.txt.omv~ shaderc-2020.4/glslc/CMakeLists.txt
--- shaderc-2020.4/glslc/CMakeLists.txt.omv~	2020-12-27 19:56:08.410601776 +0100
+++ shaderc-2020.4/glslc/CMakeLists.txt	2020-12-27 19:56:11.929653204 +0100
@@ -43,7 +43,7 @@ if (SHADERC_ENABLE_WGSL_OUTPUT)
 endif(SHADERC_ENABLE_WGSL_OUTPUT)
 
 target_link_libraries(glslc PRIVATE
-  glslang OSDependent OGLCompiler HLSL glslang SPIRV    # Glslang libraries
+  glslang HLSL glslang SPIRV    # Glslang libraries
   $<$<BOOL:${SHADERC_ENABLE_WGSL_OUTPUT}>:libtint>      # Tint libraries, optional
   shaderc_util shaderc                                  # internal Shaderc libraries
   ${CMAKE_THREAD_LIBS_INIT})
diff -up shaderc-2020.4/libshaderc/CMakeLists.txt.omv~ shaderc-2020.4/libshaderc/CMakeLists.txt
--- shaderc-2020.4/libshaderc/CMakeLists.txt.omv~	2020-12-27 19:56:21.294790003 +0100
+++ shaderc-2020.4/libshaderc/CMakeLists.txt	2020-12-27 19:56:26.648868169 +0100
@@ -62,7 +62,7 @@ endif(SHADERC_ENABLE_INSTALL)
 
 find_package(Threads)
 set(SHADERC_LIBS
-  glslang OSDependent OGLCompiler glslang ${CMAKE_THREAD_LIBS_INIT}
+  glslang ${CMAKE_THREAD_LIBS_INIT}
   shaderc_util
   SPIRV # from glslang
   SPIRV-Tools
diff -up shaderc-2020.4/libshaderc_util/CMakeLists.txt.omv~ shaderc-2020.4/libshaderc_util/CMakeLists.txt
--- shaderc-2020.4/libshaderc_util/CMakeLists.txt.omv~	2020-12-27 19:55:59.125466017 +0100
+++ shaderc-2020.4/libshaderc_util/CMakeLists.txt	2020-12-27 19:56:03.038523242 +0100
@@ -46,7 +46,7 @@ add_definitions(-DENABLE_HLSL)
 
 find_package(Threads)
 target_link_libraries(shaderc_util PRIVATE
-  glslang OSDependent OGLCompiler HLSL glslang SPIRV
+  glslang HLSL glslang SPIRV
   SPIRV-Tools-opt ${CMAKE_THREAD_LIBS_INIT})
 
 shaderc_add_tests(
